tests/integration/suite/daprd/serviceinvocation/http/basic.go

```diff
@@ -206,16 +219,18 @@ func (b *basic) Run(t *testing.T, ctx context.Context) {
 	for i := 0; i < 100; i++ {
 		t.Run("parallel requests", func(t *testing.T) {
 			t.Parallel()
-			reqURL := fmt.Sprintf("http://localhost:%d/v1.0/invoke/%s/method/foo", b.daprd1.HTTPPort(), b.daprd2.AppID())
-			req, err := http.NewRequestWithContext(ctx, http.MethodPost, reqURL, nil)
+			u := uuid.New().String()
```

Comment:
This is a modified test that sends a body and validates the response too. Since we're testing parallel things, this makes sure a response is correlated to a request.